python/labbox_ephys/import_to_workspace.py

```diff
@@ -75,32 +75,39 @@ def _import_le_sorting(sortings_subfeed: kp.Subfeed, le_sorting):
         }
     })
 
-def import_recording(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, recording_id: str):
+def random_id():
+    return str(uuid.uuid4())[-12:]
```

Comment:
Any reason to limit it to 12 characters? (I suppose that's still an awful lot of possible random numbers...)

Reply:
the structure of uuid4 is there is a group of 12 characters at the end. There won't be any penalty for changing this later if we need to. I don't know much about uuid4, so it would be good to know if this is a reasonable thing to do.

Reply:
I spent some time this morning reading up on it. It looks like the conclusion is that depending on implementation, the tail end of the UUID is likely to be the 'most random', but obviously decreasing the overall bit count increases the chances of getting a collision (even if that chance is pretty low for a 12-digit number).

That said, these IDs aren't really meant to be human-readable, so I'm not sure there's much benefit to limiting it (unless we were trying to avoid having non-numeric characters)?

Reply:
The reason to limit it is that these go in the URL for labbox-ephys. sqrt(16^12) is around 16 million. And these are specific to the given workspace, so I think we're pretty safe. Worst that can happen (very rare) is you get an error on import, and then just try again.